### ISSUE TITLE
Unused Classes:  class MapViewController {   & class MapView

### ISSUE THREAD

**Issue by siramsay** [chishiki/perihelion#13]
No way to reach the path /map

Maybe related to some of the satellites, could be moved to a module if so?

https://github.com/chishiki/perihelion/blob/main/deploy/src/perihelion/php/controller/map.view.controller.php
https://github.com/chishiki/perihelion/blob/a0ffbd66b9d69fb4bf2a53454b6f288e92ba2795/deploy/src/perihelion/php/view/map.view.php

**Comment by chishiki** [comment]
yeah this is legacy kagi code probably (e.g. https://nisekolistings.com/map/)

we should probably create a map satellite to include this code going forward

will give it some thought
